README.md

```diff
@@ -86,7 +90,11 @@ With TICSQServer, persistent measurement points are created which are stored in
 TICSQServer can also compare the last obtained results with the previous run and apply Quality Gating.
 
 ```yaml
-on: [pull_request]
+on:
+  push:
+    branches:
+      - main
+      - 'releases/**'

```

Comment:
Small question/remark: Would a more 'default-friendly' case to use for this example configuration be to run on a single branch (main) branch only? If multiple branches are included here with no TICS 'branchname' parameter configured, I believe this would cause interleaving of QServer results between the different Git branches.